# Data description  



<u>***Instructions***</u>

* *Remove instructions once you finalized the content of this file.*
* *The \Data directory is also found in the \Processing directory, and in the specific analysis directories (i.e., currently \NameOfComputation). You can use either one or all of these directories based on what you find convenient for the project. Remove the Data directories that you do not use.* 
  * *If you use multiple \Data directories then clearly describe how they relate.* 

* *Rename 'NameOfDataset_1' to a descriptive name.* 
* *Make additional directories if you have multiple datasets.*
* *In case you only have one dataset then you can remove the 'NameOfDataset_1' directory and store your data directory in \Data.*





* *The \Data directory includes:*

  * ***Raw data***
    
    * *Raw data comprise the unprocessed files that come from the machine* 
    
    * *Only unprocessed data obtained from collaborators or public databases.* 
    
    * *e.g., fastq for NGS, 'raw' for metabolomics, 'cel' for Affymetrix*
    
      
    
  * ***Meta data** (data description)*
    
    * *License*
    
    * *Description of the data obtained from collaborators*
      * *Description of samples (e.g., sample sheets)*
    
      * *Experimental design*
    
      * *Who created the data*
    
      * *Content and format of the data files*
    
      * *Why the data were generated*
    
      * *How the data were generated (measurement platform)* 
    
      * *........*
    
    * *The meta-data can also be placed in the \Raw and/or \Processed sub-directory if that is more convenient or logical.*
    
      
    
  * ***Processed data***
    
    * *Data <u>obtained from collaborators or databases</u> and, consequently, not produced as part of the your project. If data (pre)processing is part of your computational analyses then it should be placed in the \Results directory within \Processing.*
    * *Contains the non-raw data that, for example, results from pre-processing steps such as sequence alignment, normalization, summarization, and imputation (e.g., BAM for NGS, peak tables for metabolomics).* 



*FAIR and other standards*

*It is not the aim of ENCORE to FAIRify (raw, processed, meta) data obtained from collaborators or (public) databases. Nor does this data have to comply with minimum information standards, or be described using vocabularies/ontologies. In the context of ENCORE we use the data as we received.* 

* *FAIR data that complies with minimum information standards may, however, benefit the computational analysis.*

* *If  data (pre)processing is part of the project then the processing procedures should be described in detail in the \NameOfComputation directory such that it can be part of a FAIR dataset at a later stage.*



*External data*

*If, for any reason, it is impossible to store the data within the FSS then specify the location of the data, the license, and how it can be obtained.*



*Controlled access data*

*Controlled access data obtained from specific data providers (e.g., TCGA, dbGAP) should be stored  together with the complete license agreement. Controlled access cannot be shared as part of the FSS.*



*== END OF INSTRUCTIONS ==*



**Do you also use one of the other \Data directories?** [yes/no]

**How do these \Data directories relate:** [brief description]



<u>Datasets </u>

**Dataset 1:** [short description + License]



**Dataset 2:** [short description + License]



<u>External datasets</u>

**Dataset 1.**

Short description:

Reason for not including the data in this \Data directory:

How to obtain the data / Data location:

How should the data be used with the software in the \Processing directory:



 





 

